apps/web-mzima-client/src/env.json

```diff
@@ -1,6 +1,6 @@
 {
   "production": true,
-  "backend_url": "http://localhost:8080/",
+  "backend_url": "https://outreachyushahidi.api.ushahidi.io/",
   "api_v3": "api/v3/",
```

Comment:
Keep the localhost:8080 url here

Reply:
> Keep the localhost:8080 url here

Thank you @Angamanga for the review. I have make the necessary changes.

Reply:
Good day @Angamanga, This is a reminder to kindly review this changes